src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -988,6 +988,49 @@ uct_ib_mlx5_devx_md_get_counter_set_id(uct_ib_mlx5_md_t *md, uint8_t port_num)
 
 #endif
 
+/**
+ * DEVX MD API
+ */
+#if HAVE_DEVX
```

Comment:
shall we reuse the existing `#if HAVE_DEVX` above line 820?

Reply:
the existing `#if HAVE_DEVX` block defines `DEVX QP API` and have its stubs in`else` block which is not required for `MD API`, so defining it in separate section looks better to me.